src/h5web/toolbar/ComplexToolbar.tsx

```diff
@@ -55,6 +55,18 @@ function ComplexToolbar() {
 
       <Separator />
 
+      <Selector
+        value={visType}
+        onChange={(value: ComplexVisType) => setVisType(value)}
+        options={Object.values(ComplexVisType)}
+        optionComponent={({ option }) => (
+          // eslint-disable-next-line react/jsx-no-useless-fragment
+          <>{`${VIS_TYPE_SYMBOLS[option]} ${option}`}</>
+        )}
+      />
```

Comment:
Just moving this up before the ratio/grid toggles. I feel like we had discussed doing that before.